machine.go

```diff
@@ -379,7 +379,22 @@ func (m *Machine) setupscratch() error {
 	if err != nil {
 		return err
 	}
-	mkfs := exec.Command("mkfs.ext4", "-q", tmpfile.Name())
+
+	// search $PATH for executable; falling back to /sbin if not found
+	executable, err := exec.LookPath("mkfs.ext4")
+	if err != nil {
+		prefix := ""
+		if mergedUsrSystem() {
```

Comment:
no need to do that on merged systems; /sbin/mkfs will work just as well as /usr/sbin on those